modules/hcp-virt-storage-security-isolation.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-manage/hcp-manage-virt.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="hcp-virt-storage-security-isolation_{context}"]
+= KubeVirt CSI storage security and isolation
+
+KubeVirt Container Storage Interface (CSI) extends storage capabilities of the underlying infrastructure cluster to hosted clusters. The CSI driver ensures secure and isolated access to the infrastructure storage classes and hosted clusters by using the following security constraints:
```

Comment:
```suggestion
KubeVirt Container Storage Interface (CSI) extends the storage capabilities of the underlying infrastructure cluster to hosted clusters. The CSI driver ensures secure and isolated access to the infrastructure storage classes and hosted clusters by using the following security constraints:
```
On this line and elsewhere this file, should "infrastructure cluster" be "management cluster"?

Reply:
Checking on infra cluster vs management cluster.

Reply:
"management cluster and infrastructure cluster are different terms, but they can technically refer to the same cluster.
Management cluster is the cluster that the hcp control plane pods and hypershift operator run on
Infrastructure cluster is the cluster that the OpenShift Virtualization VMs run on.
The Management cluster is also the Infrastructure cluster when the VMs and HCP pods run on the same baremetal cluster"

Keeping "infrastructure cluster" as per this SME's explanation.